cedar-policy-core/src/parser/err.rs

```diff
@@ -252,8 +252,10 @@ pub enum ToASTError {
         /// The key that appeared two or more times
         key: SmolStr,
     },
-    /// Returned when a user attempts to use type-constraint syntax. This is not currently supported
-    #[error("type constraints are not currently supported")]
+    /// Returned when a user attempts to use type-constraint `:` syntax. This
+    /// syntax was not adopted, but `is` can be used to write type constraints
+    /// in the policy scope.
```

Comment:
Is there not an error for having a non-parsing EntityType after an `is`?

Reply:
That just shows up as whatever parse error was raised when the name `Failed` to parse.